tests/key_field_sample.fbs

```diff
@@ -0,0 +1,21 @@
+namespace keyfield.sample;
+
+struct Baz {
+  a: [uint8:4] (key);
```

Comment:
I would add a comment here to explain what we are testing. Something like "// A fixed-sized array of scalars as a Key"

Same with the Bar struct below